phylogenetic/defaults/config.yaml

```diff
@@ -16,8 +16,7 @@ builds: ['all-lineages']
 # See Nextstrain documentation for an explanation of how subsampling configuration works:
 # <https://docs.nextstrain.org/en/latest/guides/bioinformatics/filtering-and-subsampling.html#generalizing-subsampling-in-a-workflow>
 subsampling:
-  region: --query "is_lab_host != 'true'" --query-columns is_lab_host:str --min-length '8200' --group-by region year --subsample-max-sequences 3000 --exclude defaults/exclude.txt
-  force_include: --exclude-all --include defaults/include.txt
+  region: --query "is_lab_host != 'true'" --query-columns is_lab_host:str --min-length '8200' --group-by region year --subsample-max-sequences 3000 --exclude defaults/exclude.txt --exclude-all --include defaults/include.txt
```

Comment:
Oops, dea42d76982b9c7c0035e060767946f690abccbc incorrectly copied `--exclude-all`. It was removed in a531fd13ba9319d374c75e04cfba1cbe34c14019 so the PR's changes as whole aren't affected, but just wanted to note the bad commit.